sdk/media/azure-media-livevideoanalytics-edge/samples/sample_lva.py

```diff
@@ -0,0 +1,85 @@
+
+import json
+import os 
+from azure.media.livevideoanalytics.edge._generated.models import *
```

Comment:
Our samples should never `import` anything that is in a private/internal namespace or private/internal type, method or function (in this case, `_generated` is internal)

Reply:
(and we shouldn't do an `import *` either)